test/autoCompletion.test.ts

```diff
@@ -146,6 +146,27 @@ describe('Auto Completion Tests', () => {
           .then(done, done);
       });
 
+      it('Autocomplete on default value with \\"', async () => {
+        languageService.addSchema(SCHEMA_ID, {
+          type: 'object',
+          properties: {
+            name: {
+              type: 'string',
+              default: '\\"yaml\\"',
```

Comment:
The regex is looking for `\"`. This example does not match the video. 
It does match the expected, unless the expected resolves one slash as an escape character. I'm just not sure.

Reply:
You need to test with schema:
```json
"properties": {
  "property1": {
    "type": "string",
    "default": "\"mydefault\""
  }
}
```
In test that is not JSON, it is JS object, in JS you need to escape `\` in string literals, so if I want to have `\"` in string literal I need to have `\\"`.

Reply:
That explains my confusion. Thanks.

Reply:
No, I was wrong, it `eslint` requires to double escape `\` as it complains about unnecessary escaping. But anyway the result is the same. As for test I really need to have `\"some\"` string

Reply:
that makes me feel better

Reply:
I found a better way

Reply:
That keeps it more accurate. Nicely done.

Reply:
Yeah the new way makes a bit more sense!